src/test/java/org/kohsuke/github/GHEventPayloadTest.java

```diff
@@ -405,7 +405,6 @@ public void InstallationRepositoriesEvent() throws Exception {
         assertThat(event.getRepositorySelection(), is("selected"));
 
         assertThat(event.getRepositoriesAdded().get(0).getId(), is(186853007L));
-        assertThat(event.getRepositoriesAdded().get(0).getNodeId(), is("MDEwOlJlcG9zaXRvcnkxODY4NTMwMDc="));
```

Comment:
Why remove these?

Reply:
The previous `GHRepository` didn't have `node_id` property. Now I added it.